wsd/DocumentBroker.cpp

```diff
@@ -1764,11 +1777,10 @@ void DocumentBroker::asyncInstallPresets(const std::shared_ptr<ClientSession>& s
             std::string settings;
             if (FileUtil::Stat(viewSettings).exists())
             {
-                settings =
-                    extractViewSettings(viewSettings, session, _isViewSettingsAccessibilityEnabled);
+                settings = extractViewSettings(viewSettings, session, _isViewSettingsAccessibilityEnabled, _isViewSettingsUpdated);
                 session->sendTextFrame("viewsetting: " + settings);
+                forwardToChild(session, "addconfig");
```

Comment:
This line was moved inside the if (... viewSettings exists ...) conditional. I think if you do that then if the user doesn't have any specific view settings, but does have personal autotext/autocorr etc then they won't get used. I feel that "addconfig" line should stay where it was. Unless you have a counter reason?

Reply:
yeah, next commit puts it where I wanted it, thanks.